rules/upgrade-your-laptop/rule.md

```diff
@@ -0,0 +1,27 @@
+---
+type: rule
+title: Do you know when to upgrade your laptop?
+uri: upgrade-your-laptop
+authors:
+  - title: Ash Anil
+    url: https://www.ssw.com.au/people/ash
+created: 2022-09-27T05:52:37.565Z
+guid: 52b3f2c6-fcb8-42f6-b469-f1621f839b49
+---
+Electronic gadgets should have an expiry date because their performance fades as we use them frequently.
+
+For example, laptops are the IT employee's best friend which helps to code and make life easier. We spend thousands of $ and get a laptop with high specs and think it would be good for the next 10 years or so. 
+
+The problem with these laptops or any electronic devices (mobile) are that, as we use them the processor gets weaker and doesn't give the same performance as when it was new. 
+
+Most experts estimate a laptop’s lifespan to be three to five years. It may survive longer than that, but its utility will be limited as the components become less capable of running advanced applications. https://www.empowerit.com.au/blog/it-planning/budgeting/laptops-desktops-how-long-do-they-last/ 
+
+To keep track of the laptop's lifespan, we can use asset management tools like snipe https://snipeitapp.com/ and use the audit option to set a date after 5 years (recommended) to notify us to upgrade the laptop 
+
+![❌Figure: Bad example - Old laptops pilled up in server room](microsoftteams-image-7-.png)
+
+These are all assets tagged in Snipe (asset management) and older than 5 years. 
+
+We have a policy to get rid of laptops older than 5 years by selling them on eBay or if not working E-waste 
```

Comment:
I would turn this into a different section under a heading like "What do do with an old laptop?", I would also add a heading higher up to say "How long do laptops generally last?"